crates/solidity/testing/sanctuary/src/tests.rs

```diff
@@ -59,26 +60,40 @@ pub(crate) fn select_tests<'d>(
 }
 
 pub fn run_test(file: &SourceFile, events: &Events) -> Result<()> {
+    if !file.path.exists() {
+        // Index can be out of date:
+        events.test(TestOutcome::NotFound);
+        return Ok(());
+    }
+
+    // Ignore contracts that rely on obvious, exotic parser bugs fixed in later versions:
+    if uses_exotic_parser_bug(&file.path) {
+        events.test(TestOutcome::Incompatible);
+        return Ok(());
+    }
+
     let Some(version) = extract_compiler_version(&file.compiler) else {
         events.test(TestOutcome::Incompatible);
         return Ok(());
     };
 
-    if !file.path.exists() {
-        // Index can be out of date:
-        events.test(TestOutcome::NotFound);
+    let source = file.path.read_to_string()?;
+    // Heuristic: ignore wrongly scraped sanctuary files that
+    // contain HTML with a Cloudflare email obfuscation attribute:
```

Comment:
I suggest filing an issue at https://github.com/tintinweb/smart-contract-sanctuary with the broken contract, and adding the issue here.

Reply:
- Same below

Reply:
Done, now this mentions https://github.com/tintinweb/smart-contract-sanctuary/issues/32